system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_test.clj

```diff
@@ -327,6 +327,89 @@
                   (map #(select-keys % [:Roles :ShortName])
                        (:DataCenters (:umm concept))))))))))
 
+(deftest nil-instrument-long-name-bulk-update
+    (let [concept-ids (ingest-collection-in-umm-json-format platforms-instruments-umm)
+          _ (index/wait-until-indexed)]
+      (testing "nil instrument long name find and update"
+        (let [bulk-update-body {:concept-ids concept-ids
+                                :name "TEST NAME"
+                                :update-type "FIND_AND_UPDATE"
+                                :update-field "INSTRUMENTS"
+                                :find-value {:ShortName "atm"}
+                                :update-value {:ShortName "NSIDC"
```

Comment:
It's confusing using a data center name like NSIDC as the update value for platforms and instruments. How about "SMAP" as the platform and "LVIS" as the instrument?

Reply:
Thanks. Done.